packages/node/src/handlers/test-api.ts

```diff
@@ -34,17 +34,17 @@ export async function testApi(
 
   const workerOpts: WorkerOptions = {
     cloudProvider: config.nodeSettings.cloudProvider,
-    airnodeIdShort: wallet.getAirnodeIdShort(masterHDNode),
+    airnodeAddressShort: wallet.getAirnodeAddressShort(masterHDNode.address),
```

Comment:
On line 16 it should be calling `wallet.getAirnodeWallet(config) `and here in line 37 we should be passing the address of the airnode wallet instead.

Reply:
Why is the test passing actually? o.O